focus-block/src/Components/BlockForm/BlockForm.js

```diff
@@ -127,8 +127,9 @@ class BlockForm extends Component {
 
 		switch (event.target.name) {
 			case 'title':
-				let titleLen = event.target.value.length;
-				titleValid = titleLen > 0 ? 'valid' : 'Title cannot be empty';
+				let title = event.target.value;
+                const EMPTY_FIELD_CHECK = title.trim() === "";
```

Comment:
Super nit picky:

Can we change the string check to be single quotes? I have used single quotes across the entire app!

Reply:
2nd Note:

To keep the error style the same how about creating a helper function that checks to see if the title is valid or not based off of these params? So for example:
```
titleIsValid = (title) => {
    return (title.trim() !== '' ? 'valid' : 'Title cannot be empty');
}

...

titleValid = self.isTitleValid(event.target.value);
```
Let me know your thoughts on this!

Reply:
Hey @frubesss !

Of course, I appreciate you taking the time to contribute!

Oh my bad! Yes please use that same concept to write a validation method for the time selector. If you can find a way to create one function for ALL the checks, that would be super clean as well :)